src/polymer/iron-flex-layout-classes.ts

```diff
@@ -0,0 +1,247 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as dom5 from 'dom5';
+import {treeAdapters} from 'parse5';
+import {Document, ParsedHtmlDocument, Severity} from 'polymer-analyzer';
+
+import {HtmlRule} from '../html/rule';
+import {registry} from '../registry';
+import {FixableWarning} from '../warning';
+
+import stripIndent = require('strip-indent');
+
+import cssWhat = require('css-what');
+
+const p = dom5.predicates;
+
+const styleModules = [{
+  name: 'iron-flex',
+  selector: elementSelectorToPredicate('.layout.horizontal, .layout.vertical, .layout.inline, .layout.wrap, .layout.no-wrap, .layout.center, .layout.center-center, .layout.center-justified, .flex, .flex-auto, .flex-none')
```

Comment:
please split to 80 cols. Maybe something like:

```javascript
'.layout.horizontal, .layout.vertical, .layout.inline, .layout.wrap,' + 
' .layout.no-wrap, .layout.center, .layout.center-center, .layout.center-justified, ' +
'.flex, .flex-auto, .flex-none'
```

Reply:
Also you can run `npm run format` to run the auto-formatter over the project. (or get it integrated into your editor on save, it's clang-format)